Styling/README.md

```diff
@@ -46,6 +46,7 @@ Contents:
    * [Variables](#variables)
    * [Built-in Variables](#built-in-variables)
    * [Built-in Functions](#built-in-functions)
+   * [Batch Table Hierarchy](#batch-table-hierarchy)
```

Comment:
I don't think the TOC indentation is right here; a lot of this should not be under **Expressions**.  Can you reorganize?